### ISSUE TITLE
Added semicolon to the end of handlebars variables in CSS

### ISSUE THREAD

**Issue by isqua** [csscomb/vim-csscomb#7]
In one of my projects I have a construction like that:
```css
background: "{{background}}";
```
As you see, there is a handlebars variable. When I exes `:CSScomb` I get following result:
```css
background: "{{background;}}";
```
Is it possible to fix this behavior?

**Comment by faceleg** [comment]
This should be closed, @tonyganch?

**Comment by tonyganch** [comment]
Fixed with #9
